flake.nix

```diff
@@ -305,13 +305,20 @@
         paths = [ pkgs.openocd bscan_spi_bitstreams-pkg ];
       };
 
+      sphinx-artiq-manual = with pkgs.python3Packages; [
+            sphinx sphinx_rtd_theme
+            sphinx-argparse sphinxcontrib-wavedrom
+      ];
+
       latex-artiq-manual = pkgs.texlive.combine {
         inherit (pkgs.texlive)
           scheme-basic latexmk cmap collection-fontsrecommended fncychap
           titlesec tabulary varwidth framed fancyvrb float wrapfig parskip
           upquote capt-of needspace etoolbox booktabs;
       };
 
+      artiq-manual = [sphinx-artiq-manual latex-artiq-manual ];
```

Comment:
style (space between [ and sphinx-artiq-manual), but also I'm not sure if it's a good idea to bundle dependencies into a variable like that? That is, we don't do it anywhere else.

Reply:
Oh, hmm, is there a difference between this and grouping together dependencies into `sphinx-artiq-manual` and `latex-artiq-manual`?